src/fs/copy.js

```diff
@@ -1,5 +1,17 @@
+import fs from "fs";
+
 const copy = async () => {
-    // Write your code here 
+  const dirPath = './files'
+  const newDirPath = './files_copy'
+
+  try {
+    await fs.promises.access(dirPath)
```

Comment:
This could be moved as well, just import it then. I mean a separate module like 'utils' with functions which are common and reusable.